tests/Twig/Extension/SonataAdminExtensionTest.php

```diff
@@ -2959,10 +2973,24 @@ public function testCanonicalizedLocaleForMoment(?string $expected, string $orig
 
     public function testIsGrantedAffirmative(): void
     {
+        $this->securityChecker
```

Comment:
This has been moved where it is actually used and I changed a little because before:

```php
$this->securityChecker->isGranted(['foo', 'bar'], null)->willReturn(false);	
$this->securityChecker->isGranted(Argument::type('string'), null)->willReturn(true);
```
The first line was wrong because the implementation calls `isGranted` always with just one `$role`.